src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/PaymentOrder/Standard/StandardPaymentOrderAuthorizationTests.cs

```diff
@@ -1,103 +1,105 @@
-                .Actions.Rows[y => y.Name.Value.Contains(PaymentOrderResourceOperations.PaidPaymentOrder)].Should.BeVisible()
-                .Actions.Rows.Count.Should.Equal(3);
-
-            var order = await SwedbankPayClient.PaymentOrders.Get(orderLink, SwedbankPay.Sdk.PaymentOrders.PaymentOrderExpand.All);
-
-            // Global Order
-            Assert.That(order.PaymentOrderResponse.Amount.Value, Is.EqualTo(products.Select(x => x.UnitPrice * x.Quantity).Sum()));
-            Assert.That(order.PaymentOrderResponse.Currency.ToString(), Is.EqualTo("SEK"));
-            Assert.That(order.PaymentOrderResponse.State, Is.EqualTo(State.Ready));
-
-            // Operations
-            Assert.That(order.Operations[LinkRelation.CreatePaymentOrderReversal], Is.Null);
-            Assert.That(order.Operations[LinkRelation.CreateCancellation], Is.Not.Null);
-            Assert.That(order.Operations[LinkRelation.CreatePaymentOrderCapture], Is.Not.Null);
-            Assert.That(order.Operations[LinkRelation.PaidPaymentOrder], Is.Not.Null);
-
-            // Transactions
-            Assert.That(order.PaymentOrderResponse.CurrentPayment.Payment.Transactions.TransactionList.Count, Is.EqualTo(2));
-            Assert.That(order.PaymentOrderResponse.CurrentPayment.Payment.Transactions.TransactionList.First(x => x.Type == TransactionType.Initialization).State,
-                        Is.EqualTo(State.Completed));
-            Assert.That(order.PaymentOrderResponse.CurrentPayment.Payment.Transactions.TransactionList.First(x => x.Type == TransactionType.Authorization).State,
-                        Is.EqualTo(State.Completed));
-
-            // Order Items
-            Assert.That(order.PaymentOrderResponse.OrderItems.OrderItemList.Count, Is.EqualTo(products.Count()));
-            for (var i = 0; i < products.Count(); i++)
-            {
-                Assert.That(order.PaymentOrderResponse.OrderItems.OrderItemList.ElementAt(i).Name, Is.EqualTo(products[i].Name));
-                Assert.That(order.PaymentOrderResponse.OrderItems.OrderItemList.ElementAt(i).UnitPrice.Value, Is.EqualTo(products[i].UnitPrice));
-                Assert.That(order.PaymentOrderResponse.OrderItems.OrderItemList.ElementAt(i).Quantity, Is.EqualTo(products[i].Quantity));
-                Assert.That(order.PaymentOrderResponse.OrderItems.OrderItemList.ElementAt(i).Amount.Value, Is.EqualTo(products[i].UnitPrice * products[i].Quantity));
-            }
-        }
-
-    }
-}
+﻿using Atata;
+using NUnit.Framework;
+using Sample.AspNetCore.SystemTests.Services;
+using Sample.AspNetCore.SystemTests.Test.Helpers;
+using SwedbankPay.Sdk;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Sample.AspNetCore.SystemTests.Test.PaymentTests.PaymentOrder.Standard
+{
+    public class StandardPaymentOrderAuthorizationTests : Base.PaymentTests
+    {
+        public StandardPaymentOrderAuthorizationTests(string driverAlias)
+            : base()

```

Comment:
Codacy found an issue: [Remove this redundant 'base()' call.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5523315)